### ISSUE TITLE
Update CoW's Subgraph generated files once it's up again

### ISSUE THREAD

**Issue by ribeirojose** [cowdao-grants/cow-py#16]
CoW's hosted service subgraph is down so will add another issue to come back to fixing it and unblock the other parts of this PR

_Originally posted by @ribeirojose in https://github.com/cowdao-grants/cow-py/issues/13#issuecomment-2341056940_
